website/src/pages/index.js

```diff
@@ -82,6 +82,14 @@ function Home() {
                   Linear genome view React component on <tt>npm</tt>
                 </Link>
               </li>
+              <li>
+                <Link
+                  href="https://elliothershberg.github.io/JBrowseR/"
```

Comment:
should be gmod.github.io?

Reply:
Yeah oops, that link needs to change that link too